core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -69,7 +69,10 @@ private String writeTextOutput(StringBuilder output, PlanRepresentation plan, In
     {
         output.append(indent.nodeIndent())
                 .append(node.getName())
-                .append(node.getIdentifier())
+                .append(node.getIdentifier().entrySet().stream()
+                        .filter(entry -> !entry.getValue().isEmpty())
+                        .map(entry -> entry.getKey() + " = " + entry.getValue())
```

Comment:
Maybe convert the key from camel case to snake case (using CaseFormat) for consistency with the rest of the text format plan

Reply:
I don't think we are using snake case in text format.

Reply:
Okay, we can skip this.